SimpleDrive.App/ClientApp/components/FetchData.tsx

```diff
@@ -1,6 +1,9 @@
 import * as React from 'react';
 import { RouteComponentProps } from 'react-router';
-import 'isomorphic-fetch';
+import authService from '../services/authentication.service';
```

Comment:
`CamelCase`

Reply:
`AuthenticationService.ts`

Reply:
CamelCase is useless here - singleton is exported as service

Fixed in https://github.com/SimpleDriveOrganization/SimpleDrive/pull/14/commits/ed82b6545cfea2abe971dd85811ed9519b10e325